master/introduction/network-architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Network Architecture
+---
+
+The following overview is only valid for **container** deployments:
+
+![network architecture](/images/container-networking.layout.png)
+
+Each container is given a veth pair, with one end in the host namespace and the other
```

Comment:
I know of a 10-slide doc on it, but feels a bit overkill at this stage. I think I don't feel the container lifecycle is "dangling" as much as you do.